test/ImageSpec.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ReactTestUtils from 'react/lib/ReactTestUtils';
+import Image from '../src/Image';
+
+describe('Image', function() {
+
+  it('should be an image', function() {
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Image />
+    );
+    let image = React.findDOMNode(instance);
+
+    image.nodeName.should.equal('IMG');
+  });
+
+  it('should provide src p', function() {
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Image src="image.jpg" />
+    );
+    let image = ReactTestUtils.findRenderedDOMComponentWithTag(instance, 'IMG');
+
+    image.props.src.should.equal('image.jpg');
```

Comment:
I believe he means actually check the DOM node to see if it rendered a node with a src attribute vs checking the prop. otherwise you are just testing if react sets props instead of if you used it correctly